benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -284,7 +284,7 @@ private QueryExecutionResult executeSingleQuery(
         QueryExecutionResult result;
         macroService.runBenchmarkMacros(benchmark.getBeforeExecutionMacros(), benchmark, connection);
 
-        if (!warmup) {
+        if (!warmup && !benchmark.isThroughputTest()) {
```

Comment:
Why wouldn't we want to report status for throughput test ? It deserves a code comment

Reply:
Because this is how throughput a test was implemented. The througput test has a 
[dedicated way](https://github.com/trinodb/benchto/blob/8051535c02e47a09d0b352d1d33f55492e9c7255/benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java#L212) to save results, especially to avoid a lot overhead of saving data per query. 

Please note that https://github.com/trinodb/benchto/blob/8051535c02e47a09d0b352d1d33f55492e9c7255/benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java#L252 `warmup` was hardoded to the `false` so the behaviour was not changed by my code. It was just fixed.